/**:
  ros__parameters:
    window_name: 'ROV Camera Feed'
    publish_rate: 50.0
    deadzone: 0.1
    max_power: 1.0
    display_rate: 2.0
